scripts/mkcloud

```diff
@@ -1274,6 +1274,8 @@ Optional
         /root/.ssh/authorized_keys file and to provisioner barclamp
     want_neutronsles12=1 (default 0)
         if there is a SLE12 node, deploy neutron-l3 role into the SLE12 node
+    want_dvr=1 (default='')
```

Comment:
As I'm doing -n test, this looks correct to me... and is also used with other options, yes